reference/conanfile/methods.rst

```diff
@@ -43,6 +44,7 @@ What follows is a list of methods that you can define in your recipes to customi
 - :doc:`deploy() <methods/deploy>`: Deploys (copy from package to user folder) the desired artifacts
 - :doc:`export() <methods/export>`: Copies files that are part of the recipe
 - :doc:`export_sources() <methods/export_sources>`: Copies files that are part of the recipe sources
+- :doc:`finalize() <methods/finalize>`: Customizes the package for using it in the running machine without affecting the original package
```

Comment:
Or something more in line with "Allows to isolate user-specific configurations..."?

Reply:
The current line looks good to me.